create_jenkins_job.py

```diff
@@ -46,6 +46,14 @@
     [os.path.join(os.path.abspath(os.path.dirname(__file__)), 'job_templates')]
 
 
+def nonnegative_int(inval):
+    ret = int(inval)
+    if ret < 0:
+        # The error message here gets completely swallowed by argparse
```

Comment:
This comment should make you suspicious that something is wrong. Please check the `argparse` [docs](https://docs.python.org/3.8/library/argparse.html#type) about `type` and `ArgumentTypeError`.

Reply:
Fixed in f13827d